app/views/shared/activities/_activity_summary.html.haml

```diff
@@ -45,3 +45,25 @@
       %dd.govuk-summary-list__actions
         - if policy(activity_presenter).update_linked_activity?
           = a11y_action_link(t("default.link.#{activity_presenter.call_to_action(:linked_activity)}"), activity_step_path(activity_presenter, :linked_activity), t("activerecord.attributes.activity.linked_activity"))
+
+- if activity_presenter.title.blank?
+  .govuk-grid-row
+    .govuk-grid-column-two-thirds
+      %details.govuk-details{"data-module" => "govuk-details"}
+        %summary.govuk-details__summary
+          %span.govuk-details__summary-text
+            = t("action.activity.delete.details.summary")
+        .govuk-details__text
+          - if policy(activity_presenter).destroy?
+            %p
+              = t("action.activity.delete.details.details.authorised.explanation")
```

Comment:
Yeah, I spotted that when tinkering yesterday. Going by the class names `govuk-details__summary` and `govuk-details__text`, it would be `details.text`, but that feels too generic. What about this?

```yaml
details:
  question: This activity has no title. Can we delete it?
  answer:
    authorised: You can delete this activity as it has no title.
    authorised_action: Delete this activity
    unauthorised_html: Only BEIS can delete this activity...
```